.github/workflows/ci-python-sdk.yaml

```diff
@@ -132,6 +135,14 @@ jobs:
           --name postgres
         ports:
           - 5432:5432
+      ftp:
+        image: ghcr.io/astronomer/astro-sdk/ftp_docker
+        ports:
+          - 21:21
+          - 21000-21010:21000-21010
+        env:
+          USERS: "foo|pass|/home/foo"
```

Comment:
please make sure secret is not hardcoded

Reply:
Changed and its picked from secrets now